so-widgets-bundle.php

```diff
@@ -351,6 +356,59 @@ function admin_ajax_manage_handler(){
 	}
 
 	/**
+	 *
+	 */
+	function admin_ajax_settings_form(){
+		$widget_objects = $this->get_widget_objects();
+		$widget_object = !empty( $widget_objects[ $_GET['id'] ] ) ? $widget_objects[ $_GET['id'] ] : false;
+
+		if( empty( $widget_object ) ) exit();
+
+		$form_options = $widget_object->initialize_settings_form();
```

Comment:
Good point. On line 388 we'd still need to pass in the form options, but we could just use a line line

```
$widget_object->form( $value, $widget_object->initialize_settings_form() )
```

Alternatively, we could have a second function called `settings_form( $value )`, which would just be a wrapper to that call. That actually feels the neatest to me.